__tests__/app.test.js

```diff
@@ -15,4 +25,27 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
+});
+
+describe("GET /api/topics", () => {
+  test("200: Responds with a status code", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+  });
+
+  test("200: Responds with an array of topics.", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({body}) => {
+        const {topics} = body
+        topics.forEach((topic) => {
```

Comment:
What would happen if something happened and `topics` was sent back as an empty array `[]`? Would our test fail?